src/bosh-dns/acceptance_tests/dns-acceptance-release/jobs/configures-handler/spec

```diff
@@ -0,0 +1,14 @@
+---
+name: configures-handler
```

Comment:
whos going to use this job?

Reply:
This job is acceptance release job used in acceptance tests to prove the functionality. This is not a production release job.